mongomock/__version__.py

```diff
@@ -1,3 +1,10 @@
-import pkg_resources
+from platform import python_version_tuple
 
-__version__ = pkg_resources.get_distribution('mongomock').version
+python_version = python_version_tuple()
+
+if (int(python_version[0]), and int(python_version[1])) >= (3, 8):
```

Comment:
I think you forgot to remove the "and".
```suggestion
if (int(python_version[0]), int(python_version[1])) >= (3, 8):
```

Reply:
Whoops, thanks